fast64_internal/oot/tools/quick_import.py

```diff
@@ -66,24 +79,22 @@ def quick_import_exec(context: bpy.types.Context, sym_name: str):
     if len(all_found_defs) > 1:
         raise QuickImportAborted(
             f"Found definitions of {sym_name} in several files: "
-            + ", ".join(str(p.relative_to(assets_objects_dir_p)) for p in all_found_defs.keys())
+            + ", ".join(str(p.relative_to(found_dir_p)) for p in all_found_defs.keys())
         )
     assert len(all_found_defs) == 1
     sym_file_p, sym_defs = list(all_found_defs.items())[0]
     if len(sym_defs) > 1:
-        raise QuickImportAborted(
-            f"Found several definitions of {sym_name} in {sym_file_p.relative_to(assets_objects_dir_p)}"
-        )
+        raise QuickImportAborted(f"Found several definitions of {sym_name} in {sym_file_p.relative_to(found_dir_p)}")
 
     # We found a single definition of the symbol
     sym_def_type, sym_def_array_decl = sym_defs[0]
     is_array = sym_def_array_decl != ""
-    object_name = sym_file_p.relative_to(assets_objects_dir_p).parts[0]
+    folder_name = sym_file_p.relative_to(found_dir_p).parts[0]
 
     if sym_def_type == "Gfx" and is_array:
         settings: OOTDLImportSettings = context.scene.fast64.oot.DLImportSettings
         settings.name = sym_name
-        settings.folder = object_name
+        settings.folder = folder_name
```

Comment:
this expects an object name afaik, it wouldn't work for importing a dlist from a scene

I think this operator should be extended to more than objects if it can, but not sure how

Reply:
I renamed that because it would be weird to use ``object_name`` for cutscenes since they're in scenes, that's why I chose ``folder_name`` since it sets ``settings.folder``, but yeah tbh idk how to name that properly, that was the best idea I could find at the moment

Reply:
so, so far this can quick-import
`Gfx[]`
`(Flex)SkeletonHeader`
`AnimationHeader`
this PR adds:
`CutsceneData`

afaik the first three can only be imported by fast64 from object files (at least using the interface it gives)
so probably the code should behave different based on where it found the symbol.
e.g. if it finds a dlist in a room file, it can error with `Can't import Gfx[] from maps`, but if it's in an object then it would be ok.